src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/fakta/BehandlingFaktaService.kt

```diff
@@ -59,7 +87,17 @@ class BehandlingFaktaService(
             },
         )
 
-    private fun mapHovedytelse(søknad: SøknadBarnetilsyn?) =
+    private fun mapHovedytelseTilsynBarn(søknad: SøknadBarnetilsyn?) =
```

Comment:
Hvis man sender med `it.data.hovedytelse` til denne, så kan 1 metod dekke behovet til begge ytelser då hovedytelse-classen er gjenbrukt

Reply:
Fixed: [150794e](https://github.com/navikt/tilleggsstonader-sak/pull/452/commits/150794e1057197345bb602dd028b46a4b50a5c2d)